release-notes/EPM/I-DOCSIS/EPM_7.0.0_I-DOCSIS.md

```diff
@@ -0,0 +1,125 @@
+---
+uid: EPM_7.0.0_I-DOCSIS
+---
+
+# EPM 7.0.0 I-DOCSIS
+
+## New features
+
+## Changes
+
+### Enhancements
+
+#### CISCO CBR-8 CCAP Platform: Toggle button to enable/disable polling of OFDM/OFDMA tables [ID_36635]
+
+In some cases, it can occur that CCAP elements using the Cisco CBR-8 CCAP Platform are not able to retrieve IDs when polling the OFDM/OFDMA tables, and DataMiner interprets this as a timeout.
+
+As such, it is now possible to enable or disable the polling of these tables with a new toggle button on the *Configuration* page, so that you can prevent such timeouts.
+
+#### Only active channels associated with cable modem shown [ID_37376]
+
+When you view all channels associated with a cable modem, now only the active channels will be shown instead of all potentially available channels. This will make it easier to focus on the relevant cable modem KPIs.
+
+#### Visual pages now include parent shapes for enhanced network topology [ID_37592]
+
+Alignment errors on the Visual pages for Port, Distribution, and FAT (Forwarding Access Terminal) pages have been corrected. These Visual pages now include parent shapes, making it possible to identify the hub and the Optical Line Terminal (OLT) associated with each entity.
+
+#### Visual enhancements to EPM entity tables [ID_37776]
+
+The column order for the EPM entity tables in the *Below this view* section has been adjusted to align with the column format of the *Data* section for that same entity.
+
+#### Improved performance when navigating to EPM cards through Topology app [ID_38194]
+
+Performance has improved when you navigate to EPM cards through the Topology app. The Visual pages will now load faster and KPIs will no longer appear one at a time.
+
+#### Generic DOCSIS CM Collector now supports both comma and period as decimal separator [ID_38195]
+
+The Generic DOCSIS CM Collector connector can now parse numeric values regardless of whether a comma or period is used as decimal separator (invariant culture).
+
+#### Generic DOCSIS CM Collector: Improved calculation method for FEC values [ID_38196]
+
+The calculation method for Forward Error Correction (FEC) values in the Generic CM Collector was enhanced. Previously, FEC values were calculated using averages of channel instances, leading to potential discrepancies, particularly in scenarios where Uncorrectable Packet Ratio (UER) was high while the Corrected Packet Ratio (CER) showed as zero. Now, instead of relying on averages, the system aggregates all counters to accurately determine the ratio.
```

Comment:
```suggestion
The calculation method for Forward Error Correction (FEC) values in the Generic CM Collector has been enhanced. Previously, FEC values were calculated using averages of channel instances, leading to potential discrepancies, particularly in scenarios where Uncorrectable Packet Ratio (UER) was high while the Corrected Packet Ratio (CER) showed as zero. Now, instead of relying on averages, the system aggregates all counters to accurately determine the ratio.
```